app/hypercerts/new/page.tsx

```diff
@@ -21,7 +21,19 @@ export default async function NewHypercertPage({
       return <div>Blueprint not found</div>;
     }
 
-    formValues = fetchedBlueprint.form_values as HypercertFormValues;
+    formValues = {
```

Comment:
There's some duplication with `getDefaultFormValues()` in `components/hypercert/hypercert-minting-form/index.tsx`. Maybe that's totally fine, just wanted to check if you've seen that.

Reply:
Yes, I'm aware, difference being that getDefaultFormValues() expects a JSON string. We could reuse it but it would require stringifying fetchedBlueprint.form_values first. Some duplication until a more general pattern emerged (or not) seemed better to me.

Reply:
Yeah fully agree. We don't need to dogmatically DRY up code.